File: dataset_reduzido/codenet/Java/Code/s500433274.java
import java.math.*;
import java.util.Scanner;
public class Main {

	public static void main(String[] args) {
		BigInteger a,b,c;
		Scanner cin = new Scanner(System.in);
		while(cin.hasNext()) {
			a = cin.nextBigInteger();
			b = cin.nextBigInteger();
			c = a.max(b);
			if(a.equals(c)&&!b.equals(c)) 
				System.out.println("GREATER");
			else if(b.equals(c)&&!a.equals(c))
				System.out.println("LESS");
			else 
				System.out.println("EQUAL");
		}
		cin.close();
	}

}

File: output/Llama-2-100/codenet/Java/Python/s500433274.py
import math
import input

def main():

a,b,c = input.nextBigInteger(), input.nextBigInteger(), max(a,b)

if a == c and not b == c:
    print("GREATER")
elif b == c and not a == c:
    print("LESS")
else:
    print("EQUAL")

input.close()
